nncf/common/pruning/utils.py

```diff
@@ -387,11 +389,14 @@ def get_input_channels(node: NNCFNode) -> int:
     :param node: Given prunable node.
     :return: Count of input channels of the given node.
     """
-    layer_attrs = node.layer_attributes # type: Union[ConvolutionLayerAttributes, LinearLayerAttributes]
+    layer_attrs = node.layer_attributes \
+        # type: Union[ConvolutionLayerAttributes, LinearLayerAttributes, EmbeddingLayerAttributes]
```

Comment:
It seems like a useless type hint for me that just creates a noise.
@daniil-lyakhov do you remember why do we need it?

Reply:
`node.layer_attributes` is not an input parameter. But it can be mentioned in the description of `graph`.